docs/performance/mobile-performance-tuning.md

```diff
@@ -14,16 +16,14 @@ ONNX Runtime Mobile can be used to execute ORT format models using NNAPI (via th
 
 First, please review the introductory details in [using NNAPI with ONNX Runtime Mobile](../execution-providers/NNAPI-ExecutionProvider.md) and [using CoreML with ONNX Runtime](../execution-providers/CoreML-ExecutionProvider.md).
 
-**IMPORTANT NOTE:** The examples on this page refer to the NNAPI EP for brevity. The information equally applies to the CoreML EP, so any reference to 'NNAPI' below can be substituted with 'CoreML'.<br>
-Support for creating a CoreML-aware ORT format model, similar to [creating an NNAPI-aware ORT format model](#3-creating-an-nnapi-aware-ort-format-model), was added in ONNX Runtime version 1.9.
+**IMPORTANT NOTE:** The examples on this page refer to the NNAPI EP for brevity. The information equally applies to the CoreML EP, so any reference to 'NNAPI' below can be substituted with 'CoreML'.
 
 ## Contents
 {: .no_toc}
 
 * TOC
 {:toc}
 
-
 ## 1. ONNX Model Optimization Example
```

Comment:
nit: now that the model conversion script doesn't require specifying the optimization level, this feels somewhat disconnected from the 'runtime' vs 'full' parameters that a user will probably encounter first. if they're new to ORT they will have no idea about optimization levels.

Would it be simpler for a new user to first read something explaining at a high level the why/when of 'full' vs. 'runtime' (maybe move the section on runtime optimization up), followed by the deep dive into basic vs extended optimizations and their impacts on the nodes generated and whether NNAPI can be used? rationale is that most users won't care about the lower level details after reading the high level info, so it's better to put that first.